docs/management/numeral.asciidoc

```diff
@@ -0,0 +1,175 @@
+[[numeral]]
+== Elastic Numeral Formatting
```

Comment:
Remove Elastic from the title. Instead use either of these titles:

Numeral formatting 
Numeral formatting in Kibana